README.md

```diff
@@ -281,49 +281,43 @@ Learn more about <a href="https://docs.meshery.io/architecture">Meshery's archit
 ## Join the Meshery community!
 
 <a name="contributing"></a><a name="community"></a>
-Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://layer5.io/community/handbook/repository-overview">Layer5 Community Welcome Guide</a> for a tour of resources available to you and see the <a href="https://layer5.io/community/handbook/repository-overview">Layer5 Repository Overview</a> for a cursory description of repository by technology and programming language. Jump into community <a href="http://slack.layer5.io">Slack</a> to engage!
+Our projects are community-built and welcome collaboration. 👍 Be sure to see the <a href="https://layer5.io/community/handbook/repository-overview">Community Welcome Guide</a> for a tour of resources available to you and jump into our <a href="https://slack.meshery.io">Slack</a>!
```

Comment:
The link here is not to the Community Welcome Guide. That Google Doc was deleted by a community member and cannot be recovered.

1. Remove this link and the reference to this guide, please. Replace with a link to https://layer5.io/community/newcomers and a different phrase.
1. Reinstate the phrase and link about the repo overview